src/lib/qnio/utils.c

```diff
@@ -9,6 +9,117 @@
  */
 
 #include "defs.h"
+#include "qnio.h"
+
+int 
+is_secure()
+{
+    if (access(SECURE_IMPL, F_OK) != 0)
```

Comment:
Turning on / off use of SSL based on existence of a random path on host is awful behaviour.

This needs to be under direct control via the API. ie, QEMU must be able to instruct libqnio whether or not to use SSL for each connection made, without side effects from host OS state.